smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/connectors/WorkerPoolRegistry.java

```diff
@@ -71,9 +73,26 @@ public void init() {
         }
         Objects.requireNonNull(uni, msg.actionNotProvided());
 
+        Context currentContext = Vertx.currentContext();
```

Comment:
wondering if we should not retrieve the message context.

Reply:
Indeed, I inlined the executeBlocking on the worker pool.